src/components/radio/Radio.js

```diff
@@ -161,24 +161,52 @@ export default class RadioComponent extends Field {
   }
 
   setSelectedClasses() {
+    switch (this.inputInfo.attr.type) {
```

Comment:
This can just be changed to...

```js
this.component.values.forEach(item => {
  item.selected = this.inputInfo.attr.type === 'checkbox' ? this.dataValue[item.value] : (item.value === this.dataValue);
});
```